docker/frankenphp/Caddyfile

```diff
@@ -0,0 +1,23 @@
+{
+	{$CADDY_GLOBAL_OPTIONS}
+
+	frankenphp {
+		{$FRANKENPHP_CONFIG}
+	}
+}
+
+{$SERVER_NAME:localhost} {
```

Comment:
Did you try to set `https://github.com/dunglas/frankenphp/discussions/718`

```
SERVER_NAME: :80
```


To disable https, because we will serve caddy behind a reverse proxy for SSL : i.e. Traefik

Reply:
```
servers {
		trusted_proxies static 0.0.0.0/0
	}
```

With the `trusted_proxies` directive